content/blog/2017/2017-03-16-fosdem-event-report.adoc

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title: 'FOSDEM 2017 Wrap-up'
+tags:
+- fosdem
+- event
+author: rtyler
+---
+
+
+In early February numerous free and open source developers from around the
+world traveled to Brussels, Belgium, for arguably the largest event of it's
+kind:
+link:https://fosdem.org[FOSDEM]. Among the thousands of hackers in attendance,
+were a dozen or so Jenkins contributors. We have attended the event for the
+past, but this year we had a blizzard of activity spanning four days around the FOSDEM
+weekend.
+
+.City Hall, photo by Kohsuke Kawaguchi
+image::https://c1.staticflickr.com/3/2328/32449169840_f8fec9c11a_z.jpg[Brussels City Hall, role=center, link="https://www.flickr.com/photos/12508267@N00/32449169840/in/album-72157680098353926/"]
+
+
+One of our "accidental traditions" has become a
+link:https://www.meetup.com/jenkinsmeetup/events/236370435/[happy hour]
+the Friday night before FOSDEM truly begins at Cafe Le Roy d'Espagne on Grand
+Place right in the middle of Brussels. Conveniently located a few hundred meters away from the
+link:https://fosdem.org/2017/beerevent/[FOSDEM Beer Event]
+at Delirium Cafe, each year we are inevitably joined by friends from other open
+source projects who know they're welcome to join us for a few drinks.
+
+.Cafe Le Roy, photo by Kohsuke Kawaguchi
+image::https://c1.staticflickr.com/1/497/31987111834_39773bc5ab_z.jpg[Happy hour at Cafe Le Roy, role=center, link="https://www.flickr.com/photos/12508267@N00/31987111834/in/album-72157680098353926/""]
+
+After dinner and few drinks, a few of us decided it would be a good idea (it
+wasn't) to walk over to check out the FOSDEM Beer Event and _maybe_ have just
+one more beer. For the uninitiated, Belgian beers tend to be strong, as the FOSDEM organizers warn:
+
+[quote]
+____
+Unlike some other beers, Belgian beer is not just coloured water. Some beers
+contain significant quantities of alcohol and will give you a pounding hangover
+____
+
+Unfortunately, some of us seem to re-learn this lesson each year at FOSDEM!
+
+---
+
+Bright and early the following day, FOSDEM really kicked off with keynotes and
+the project tables lining a number of corridors.
+
+.A busy hall at FOSDEM, photo by Kohsuke Kawaguchi
+image::https://c1.staticflickr.com/3/2035/32706234821_2997a98b52_z.jpg[Busy FOSDEM hall, role=center, link="https://www.flickr.com/photos/12508267@N00/32706234821/in/album-72157680098353926/""]
+
+At the Jenkins project we typically spend two full days of answering questions,
+showing off the latest and greatest Jenkins features, and of course handing out
+Jenkins stickers. The table is where many contributors, myself included, have
+a rare opportunity to talk with dozens of enthusiastic Jenkins users from
+across the open source ecosystem. This year we were very fortunate to have a
```

Comment:
To me, "ecosystem" implies the software and technology. I might be inclined to use the phrasing, "from all parts of the open source community," or something like that. But I leave it to you, this gets into your voice vs. mine.